frb_example/pure_dart/rust/src/frb_generated.io.rs

```diff
@@ -39040,2127 +36229,2117 @@ pub struct wire_cst_exotic_optionals_twin_sync_sse {
     r#type: *mut wire_cst_list_prim_u_8,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_raw_string_item_struct_twin_sse {
     r#type: *mut wire_cst_list_prim_u_8,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_raw_string_item_struct_twin_sync {
     r#type: *mut wire_cst_list_prim_u_8,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_raw_string_item_struct_twin_sync_sse {
     r#type: *mut wire_cst_list_prim_u_8,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_raw_string_mirrored {
     value: *mut wire_cst_list_prim_u_8,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_record_application_settings_raw_string_enum_mirrored {
     field0: wire_cst_application_settings,
     field1: wire_cst_raw_string_enum_mirrored,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_record_string_i_32 {
     field0: *mut wire_cst_list_prim_u_8,
     field1: i32,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_sequences {
     field0: *mut wire_cst_list_prim_i_32,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_some_struct_twin_normal {
     value: u32,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_some_struct_twin_rust_async {
     value: u32,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_some_struct_twin_rust_async_sse {
     value: u32,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_some_struct_twin_sse {
     value: u32,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_some_struct_twin_sync {
     value: u32,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_some_struct_twin_sync_sse {
     value: u32,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_speed_twin_normal {
     tag: i32,
-    kind: *mut SpeedTwinNormalKind,
+    kind: SpeedTwinNormalKind,
 }
 #[repr(C)]
+#[derive(Clone, Copy)]
 pub union SpeedTwinNormalKind {
-    Unknown: *mut wire_cst_SpeedTwinNormal_Unknown,
-    GPS: *mut wire_cst_SpeedTwinNormal_GPS,
+    GPS: core::mem::ManuallyDrop<wire_cst_SpeedTwinNormal_GPS>,
+    nil__: (),
 }
 #[repr(C)]
-#[derive(Clone)]
-pub struct wire_cst_SpeedTwinNormal_Unknown {}
-#[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_SpeedTwinNormal_GPS {
     field0: f64,
 }
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_speed_twin_rust_async {
     tag: i32,
-    kind: *mut SpeedTwinRustAsyncKind,
+    kind: SpeedTwinRustAsyncKind,
 }
 #[repr(C)]
+#[derive(Clone, Copy)]
 pub union SpeedTwinRustAsyncKind {
-    Unknown: *mut wire_cst_SpeedTwinRustAsync_Unknown,
-    GPS: *mut wire_cst_SpeedTwinRustAsync_GPS,
+    GPS: core::mem::ManuallyDrop<wire_cst_SpeedTwinRustAsync_GPS>,
```

Comment:
nit: Curious why do ManuallyDrop? We do not seem to generate drop functions manually, so will we have memory leaks?

Reply:
Actually this is (was) a requirement for union fields that I briefly mentioned earlier, that they must be either Copy or wrapped in ManuallyDrop. And they are all Copy now so we can remove this.